src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -64,9 +64,9 @@ public PersistentDataStore() {
       try {
         UUID uuid = UUID.fromString((String) entity.getProperty("uuid"));
         String userName = (String) entity.getProperty("username");
+		String password = (String)entity.getProperty("password");
```

Comment:
Tiny thing: you probably don't need this indentation?

Reply:
I have no idea why that looks like this on github.
![capture](https://user-images.githubusercontent.com/22554962/37249006-5f72c8fa-24ac-11e8-9cfb-d31979c8acf2.PNG)

Reply:
Hmmm maybe a result of using tabs instead of spaces? Either way I'm not too worried about it, so I wouldn't consider this a merge blocker 😸

Reply:
You are right! It's because I'm using tabs not spaces.